yaml/buildings.yaml

```diff
@@ -106,6 +106,9 @@ filters:
   # address points
   - filter:
       "addr:housenumber": true
+      way:
```

Comment:
Do we want to drop addresses unless they are on a building? According to [taginfo](http://taginfo.openstreetmap.org/keys/addr:housenumber#combinations), ~ 60% don't have a `building` tag.